pkgs/development/libraries/science/math/spooles/default.nix

```diff
@@ -15,6 +15,10 @@ stdenv.mkDerivation rec {
     ./spooles.patch
   ];
 
+  postPatch = stdenv.lib.optionalString stdenv.cc.isClang ''
```

Comment:
Is `-install_name` a clang thing? I thought it was a macos thing...

Reply:
Oh! I believe you are correct. Should this be hostPlatform.isDarwin then?

Reply:
I _think_ so.